index.html

```diff
@@ -0,0 +1,160 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/index.css">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css">
+
+
+    <!-- Fonts -->
+    <link href="https://fonts.googleapis.com/css?family=Lato&display=swap" rel="stylesheet">
+
+    <!-- Font Awesome -->
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.2/css/all.css"
+        integrity="sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr" crossorigin="anonymous">
+    <title>Document</title>
+</head>
+
+<body>
+
+
+    <header>
+        <div class="topnav">
+            <nav>
+                <li><a href=""><i class="fa-brands fa-instagram-square"></i></a></li>
+                <li><a href=""> <i class="fa-brands fa-twitter"></i></a></li>
+                <li><a href=""><i class="fa-brands fa-facebook"></i></a></li>
+            </nav>
+        </div>
+        <div class="mainnav">
+            <h1 class="main-h1"> The chance</h1>
+            <nav>
+                <li><a href="/about.html"> About</a> </li>
+                <li> Contact</li>
+                <li> Sponsor</li>
+                <li>News</li>
+                <button class="main-button"> Buy Ticket now</button>
+            </nav>
+        </div>
+
+
+        <button type="button" class="ham">
+            <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke="currentColor">
+                <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M4 6h16M4 12h16M4 18h16" />
+            </svg>
+        </button>
+
+
+
+        <div class="overlayy"></div>
+        <nav class="sideNav">
+            <button type="button" class="menuX">
+                <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke="currentColor">
+                    <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M6 18L18 6M6 6l12 12" />
+                </svg>
+            </button>
+            <ul>
+                <li><a class="menuLink" href="/about.html">About</a></li>
+                <li><a class="menuLink" href="#projects-container">Contact</a></li>
+                <li><a class="menuLink" href="#about-container">Sponsort</a></li>
+                <li><a class="menuLink main-button" href="#footer">Buy Tickets</a></li>
+            </ul>
+        </nav>
+    </header>
+
+    <section class="showcase">
```

Comment:
- [ ] Please put the texts and the titles a little bit to the top as there is a big space.
![Screenshot from 2022-05-19 23-22-48](https://user-images.githubusercontent.com/33752735/169407715-16f5dc31-48ed-404e-a321-9dfa5f80abb1.png)